test/model_test.dart

```diff
@@ -170,7 +170,7 @@ void main() {
 
     test('docs refs', () {
       expect(B.resolveReferences(),
-          'Extends class <a href="ex/Apple_class.html">Apple</a>');
+          'Extends class <a href="ex/Apple_class.html">Apple</a>, use new <a href="ex/Apple/Apple.html">Apple</a>');
```

Comment:
I think we should include `new` in the link itself, making it clearer that it's a constructor we're linking to